nix/docker.nix

```diff
@@ -72,12 +72,11 @@ in
       mkdir -p data config
     '';
     config = {
-      EntryPoint = [ "${cardano-node}/bin/cardano-node" "run" "--config"
-      "/config/config.yaml" "--database-path" "/data.db" "--genesis-file"
-      "/config/genesis.json" "--host-addr" "127.0.0.1" "--port" "3001"
-      "--socket-path" "/data/ipc/socket0" "--topology" "/config/topology.json"];
+      EntryPoint = [ scripts.${env.name}.node ];
```

Comment:
The foundation image was purposely free from environmental concerns, which is why the `ENTRYPOINT` was `cardano-node` with configuration options based on the file system conventions. Does this change cover the requirement and produce the same image as the previous version?

Reply:
I've added custom. It's not very user friendly. My preference would have been to have a wrapper that checked if a file existed in /config for topology, config and genesis and used that if it did, but @craigem said that doesn't match the spec you provided to him. So now you either run dockerImage.<env> or dockerImage.custom. env just works. custom you have to pass it everything (probably through a mounted volume). It's not something I'd want end users to use, and maybe we can follow up with a better script wrapper for environments to be able to override parts of an environment in a follow-up.

Reply:
I've built and tested `dockerImage.custom` and I believe that as of commit 6a5b233 it works as expected.